pkg/option/config.go

```diff
@@ -890,6 +891,8 @@ var HelpFlagSections = []FlagsSection{
 			EnableHealthChecking,
 			TracePayloadlen,
 			PProf,
+			Restore,
+			KeepConfig,
```

Comment:
These aren't to do with debug / logging / trace, what's the motivation for moving these here?

Reply:
Will fix it.

Reply:
Done.